coderwall.svg

```diff
@@ -1,58 +1,9 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!-- Created with Inkscape (http://www.inkscape.org/) -->
-
-<svg
-   xmlns:dc="http://purl.org/dc/elements/1.1/"
-   xmlns:cc="http://creativecommons.org/ns#"
-   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-   xmlns:svg="http://www.w3.org/2000/svg"
-   xmlns="http://www.w3.org/2000/svg"
-   version="1.1"
-   width="28"
-   height="27"
-   id="svg3039">
-  <metadata
-     id="metadata3045">
-    <rdf:RDF>
-      <cc:Work
-         rdf:about="">
-        <dc:format>image/svg+xml</dc:format>
-        <dc:type
-           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
-        <dc:title></dc:title>
-      </cc:Work>
-    </rdf:RDF>
-  </metadata>
-  <defs
-     id="defs3043" />
-  <path
-     d="m 10.247191,7.9550562 a 3.9101124,3.775281 0 1 1 -7.8202248,0 3.9101124,3.775281 0 1 1 7.8202248,0 z"
-     transform="matrix(0.67889237,0,0,0.70313852,-1.3021943,-1.5935064)"
-     id="path3049"
-     style="fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1" />
-  <path
-     d="m 10.247191,7.9550562 a 3.9101124,3.775281 0 1 1 -7.8202248,0 3.9101124,3.775281 0 1 1 7.8202248,0 z"
-     transform="matrix(0.67889237,0,0,0.70313852,8.2978057,-1.5935064)"
-     id="path3049-3"
-     style="fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1" />
-  <path
-     d="m 10.247191,7.9550562 a 3.9101124,3.775281 0 1 1 -7.8202248,0 3.9101124,3.775281 0 1 1 7.8202248,0 z"
-     transform="matrix(0.67889237,0,0,0.70313852,17.897806,-1.5935064)"
-     id="path3049-35"
-     style="fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1" />
-  <path
-     d="m 10.247191,7.9550562 a 3.9101124,3.775281 0 1 1 -7.8202248,0 3.9101124,3.775281 0 1 1 7.8202248,0 z"
-     transform="matrix(0.67889237,0,0,0.70313852,8.2978057,7.0064936)"
-     id="path3049-3-0"
-     style="fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1" />
-  <path
-     d="m 10.247191,7.9550562 a 3.9101124,3.775281 0 1 1 -7.8202248,0 3.9101124,3.775281 0 1 1 7.8202248,0 z"
-     transform="matrix(0.67889237,0,0,0.70313852,17.897806,7.0064936)"
-     id="path3049-35-7"
-     style="fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1" />
-  <path
-     d="m 10.247191,7.9550562 a 3.9101124,3.775281 0 1 1 -7.8202248,0 3.9101124,3.775281 0 1 1 7.8202248,0 z"
-     transform="matrix(0.67889236,0,0,0.70313852,17.897806,15.606494)"
-     id="path3049-35-6"
-     style="fill:#000000;fill-opacity:1;stroke:#000000;stroke-opacity:1" />
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+  <circle cx="5" cy="5" r="1.4142" fill="#3e8dcc"/>
```

Comment:
Analyzing the CSS from <http://codepen.io/TimPietrusky/pen/hLolB>, the entire bg has `2em` of width, and each dot is `1em`, with `0.5em` offset from each one. Multiplying all by 10, we have the `viewBox="0 0 20 20"` and each center on `5` SVG points increments.

The radius was a bit more tricky. The CSS used `20%`; but what does that percentage mean? It is measured from the center to the furthest corner, which is the diagonal of the square. `20% × 0.5 × √2 ≅ 0.14142135623`

I'm pretty sure that `20%` is just some approximation by the author of that CSS example, but all "official" logos are low-quality raster pictures anyway.